src/components/UI/Button/Button.tsx

```diff
@@ -28,6 +28,7 @@ export const Button: FunctionComponent<ButtonProps> = ({ className, children, di
         disabled ? "cursor-not-allowed bg-grey-200 text-white hover:bg-grey-200" : ""
       }`}
       type="submit"
+      disabled={disabled}
```

Comment:
oh i think u don't have to disabled here, i've already catered to https://github.com/TradeTrust/tradetrust-ui-components/blob/master/src/components/UI/Button/Button.tsx#L28

unless if u meant to disable click event 🤔

Reply:
ya i need the 'disabled' attribute for some test cases to work. And ur disable is only css disable right? so yup i think need the disable click event too.